examples/dotnet/MyStack.cs

```diff
@@ -0,0 +1,61 @@
+// Pulumi Command Provider Node SDK
```

Comment:
.NET SDK. Same for all other files.

Reply:
That's what I get for copying it too quickly! Will update.